manifests/ubuntu.pp

```diff
@@ -52,8 +56,7 @@
 
   package { 'datadog-agent':
     ensure  => $agent_version,
-    require => [File['/etc/apt/sources.list.d/datadog.list'],
-                Exec['datadog_apt-get_update']],
+    require => Exec['datadog_apt-get_update'],
```

Comment:
`Exec['datadog_apt-get_update']` will only happen if you manage the repo, so it should be dropped entirely